README.md

```diff
@@ -1 +1,85 @@
-# budget-app
\ No newline at end of file
+# Budget App
+
```

Comment:
The provided video link to this Pull Request does is invalid. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The recorded video should:
- [ ]  entail the **features** of the project.
- [ ] have a maximum length of **5 minutes**.
- [ ] be **clear and objective description** about the project in professional English. The topic and purpose of the project are made clear.
- [ ] contains a demo of the project, highlighting all the project features.
- [ ] contain a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
:+1: Good presentation.